core/src/main/scala/org/broadinstitute/dsde/rawls/model/ExecutionModel.scala

```diff
@@ -375,12 +375,13 @@ class ExecutionJsonSupport extends JsonSupport {
         // so in that project one can just adjust the function call to jsonFormat*N*.
         methodConfigurationNamespace = fields("methodConfigurationNamespace").convertTo[String],
         methodConfigurationName = fields("methodConfigurationName").convertTo[String],
-        entityType = fields.get("entityType").map(_.convertTo[String]),
-        entityName = fields.get("entityName").map(_.convertTo[String]),
-        expression = fields.get("expression").map(_.convertTo[String]),
+        // We are flatMapping these because `null` is a valid input provided by the UI
```

Comment:
Sorry I'm a little late to this, but maybe it's a bug that the UI is sending `null`s? I have not investigated anything yet but I can if you think there may be a UI problem, @aednichols .

Reply:
We're talking the FireCloud UI, which didn't change within the timespan when the bug appeared